redisvl/index.py

```diff
@@ -379,48 +315,6 @@ class SearchIndex(SearchIndexBase):
         >>> index.load(data) # data is an iterable of dictionaries
     """
 
-    @classmethod
-    def from_existing(
```

Comment:
deprecate instead?

Reply:
force it? i.e. throw a warning but allow it? Or raise an exception that forces the user's hand to change their approach? I think it's safe to break it here, but would be ok with a depwarning for sure

Reply:
ok with either. Break is fine as long as we update to 0.1.0 which is the plan